src/AppKit/NSMutableFontCollection.cs

```diff
@@ -0,0 +1,23 @@
+//
+// NSMutableFontCollection.cs
+//
+// Author:
+//  Chris Hamons <chris.hamons@xamarin.com>
+//
+// Copyright 2016 Xamarin Inc. (http://xamarin.com)
+
+
+using System;
+
+
+namespace XamCore.AppKit
+{
+#if !XAMCORE_4_0
+	public partial class NSMutableFontCollection
+	{
+		public NSMutableFontCollection ()
```

Comment:
add an `[Obsolete]` with a description so anyone using it will notice (and no one else start using it and wonder why nothing works)